### PR TITLE
Update to v8 11.4.183.12

### PR DESCRIPTION
**Pull request by ohodson**
Test: bazel test //...

### CODE REVIEW

File: docs/v8-updates.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+   There is usually some minor patch editing required during rebase.
+
+   Ideally at this stage, you should be able to build and test V8 with the patches applied.
+
+8. Re-generate workerd's v8 patches. Assuming there are 8 workerd patches for V8,
+   the command would be:
+
+   ```
+   $ git format-patch -k --no-signature HEAD~8
```
Comment (harrishancock) [inline docs/v8-updates.md:54]: May I suggest `--full-index`, to keep the full hashes in the patch files? It doesn't really matter either way, so long as everyone does the same thing (to minimize unnecessary diffs), and that's what we've standardized on internally.

**Review by harrishancock — APPROVED** [review]
Nice new v8-updates.md document -- thanks for writing it!

Non-blocking nit, but could you add guidance on how to update `shallow_since`, too? Do you just take the output of `date -u +"%s %z"`?

**Comment by ohodson** [comment]
> 

I've added a note in 82d3611975926c2c40e4b2e8d609a3e4ec5bc895. There is probably some git magic to establish that date, but bazel spits it out during the build.

**Comment by ohodson** [comment]
Folded into https://github.com/cloudflare/workerd/pull/689

Review comment suggestions that missed the folding are in https://github.com/cloudflare/workerd/pull/697

**Pull request closed without merging**